pegasus/helpers/form_helpers.rb

```diff
@@ -129,17 +131,19 @@ def insert_form(kind, data, options={})
 
   timestamp = DateTime.now
 
+  normalized_email = data[:email_s].to_s.strip.downcase
   row = {
     secret: SecureRandom.hex,
     parent_id: options[:parent_id],
-    email: data[:email_s].to_s.strip.downcase,
+    email: normalized_email,
     name: data[:name_s].to_s.strip,
     kind: kind,
     data: data.to_json,
     created_at: timestamp,
     created_ip: request.ip,
     updated_at: timestamp,
     updated_ip: request.ip,
+    hashed_email: Digest::MD5.hexdigest(normalized_email),
```

Comment:
Is this safe? Do we need to update any other places where these forms are accessed by users? Do we (presumably) intend to do a data fix on existing data?